types/coin.go

```diff
@@ -368,6 +368,23 @@ func (coins Coins) IsAllLTE(coinsB Coins) bool {
 	return coinsB.IsAllGTE(coins)
 }
 
+// IsAnyGT returns true iff for any denom in coins, the denom is present at a
+// greater amount in coinsB.
+func (coins Coins) IsAnyGT(coinsB Coins) bool {
```

Comment:
I think we can let this go now (same goes for respective test case and and pending entry)

Reply:
I know, it'll be removed.